app/src/main/java/com/soen390/conumap/ui/search_completed/SearchCompletedFragment.kt

```diff
@@ -58,8 +62,16 @@ class SearchCompletedFragment : Fragment() {
 
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
-        viewModel = ViewModelProviders.of(this).get(SearchCompletedViewModel::class.java)
```

Comment:
This is because this viewModel is not used yet and it is better for us to only use the SearchBarViewModel to communicate the data between fragments more easily
[same comment as below since same problem]